### ISSUE TITLE
如何训练可以达到权重的精度？

### ISSUE THREAD

**Issue by Runist** [woshidandan/Image-Aesthetics-Assessment#5]
这边按照默认option的参数从头训练了20个epcoh，lcc只有0.10多，是需要载入预训练权重吗？backbone是你们自己设计的，你们应该也是直接在AVA上训练的吧？

**Comment by Runist** [comment]
这版本代码里面有bug
`for idx, (x_input, y_true_prob) in enumerate(train_loader):`
这样取出来的idx会因为多线程的问题取出来的是不对的，你再用idx去算step，写入的图表也是乱的

**Comment by NurseryRhymeA** [comment]
您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？

**Comment by Runist** [comment]
> 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？

得用预训练权重，不然没有泛化性

**Comment by NurseryRhymeA** [comment]
谢谢！

**Comment by YUTING0907** [comment]
> > 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？
> 
> 得用预训练权重，不然没有泛化性

请问预训练权重文件在哪里呀？

**Comment by NurseryRhymeA** [comment]
> > > 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？
> > 
> > 
> > 得用预训练权重，不然没有泛化性
> 
> 请问预训练权重文件在哪里呀？

作者在readme里提供了

download weights from: https://drive.google.com/drive/folders/1UpLYGLU5omztVsIWkRPFTVKAOVe_4p3K?usp=sharing

我遇到了AVA数据集的预训练权重指标不太对的问题，如果您训练出结果可以提供一下您训练之后的指标吗，想做一个参考，感谢

**Comment by YUTING0907** [comment]
> > > > 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？
> > > 
> > > 
> > > 得用预训练权重，不然没有泛化性
> > 
> > 
> > 请问预训练权重文件在哪里呀？
> 
> 作者在readme里提供了
> 
> download weights from: https://drive.google.com/drive/folders/1UpLYGLU5omztVsIWkRPFTVKAOVe_4p3K?usp=sharing
> 
> 我遇到了AVA数据集的预训练权重指标不太对的问题，如果您训练出结果可以提供一下您训练之后的指标吗，想做一个参考，感谢

这里的权重应该是author训练出来的模型文件，我想复现下，但是跑了几个epoch发现指标纹丝不动，囧

**Comment by NurseryRhymeA** [comment]
> > > > > 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？
> > > > 
> > > > 
> > > > 得用预训练权重，不然没有泛化性
> > > 
> > > 
> > > 请问预训练权重文件在哪里呀？
> > 
> > 
> > 作者在readme里提供了
> > download weights from: https://drive.google.com/drive/folders/1UpLYGLU5omztVsIWkRPFTVKAOVe_4p3K?usp=sharing
> > 我遇到了AVA数据集的预训练权重指标不太对的问题，如果您训练出结果可以提供一下您训练之后的指标吗，想做一个参考，感谢
> 
> 这里的权重应该是author训练出来的模型文件，我想复现下，但是跑了几个epoch发现指标纹丝不动，囧

抱歉我记错了，在https://github.com/LeapLabTHU/DAT    
中有网络在imagenet上预训练的的模型，因为EAT网络结构变了所以读取的时候要改一下，作者也写了读取函数，不过好像要翻历史版本，现在提供的是又一次修改后的模型

**Comment by YUTING0907** [comment]
> > > > > > 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？
> > > > > 
> > > > > 
> > > > > 得用预训练权重，不然没有泛化性
> > > > 
> > > > 
> > > > 请问预训练权重文件在哪里呀？
> > > 
> > > 
> > > 作者在readme里提供了
> > > download weights from: https://drive.google.com/drive/folders/1UpLYGLU5omztVsIWkRPFTVKAOVe_4p3K?usp=sharing
> > > 我遇到了AVA数据集的预训练权重指标不太对的问题，如果您训练出结果可以提供一下您训练之后的指标吗，想做一个参考，感谢
> > 
> > 
> > 这里的权重应该是author训练出来的模型文件，我想复现下，但是跑了几个epoch发现指标纹丝不动，囧
> 
> 抱歉我记错了，在https://github.com/LeapLabTHU/DAT 中有网络在imagenet上预训练的的模型，因为EAT网络结构变了所以读取的时候要改一下，作者也写了读取函数，不过好像要翻历史版本，现在提供的是又一次修改后的模型

好滴，非常感谢，我先看看

**Comment by YUTING0907** [comment]
> > > > > > > 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？
> > > > > > 
> > > > > > 
> > > > > > 得用预训练权重，不然没有泛化性
> > > > > 
> > > > > 
> > > > > 请问预训练权重文件在哪里呀？
> > > > 
> > > > 
> > > > 作者在readme里提供了
> > > > download weights from: https://drive.google.com/drive/folders/1UpLYGLU5omztVsIWkRPFTVKAOVe_4p3K?usp=sharing
> > > > 我遇到了AVA数据集的预训练权重指标不太对的问题，如果您训练出结果可以提供一下您训练之后的指标吗，想做一个参考，感谢
> > > 
> > > 
> > > 这里的权重应该是author训练出来的模型文件，我想复现下，但是跑了几个epoch发现指标纹丝不动，囧
> > 
> > 
> > 抱歉我记错了，在https://github.com/LeapLabTHU/DAT 中有网络在imagenet上预训练的的模型，因为EAT网络结构变了所以读取的时候要改一下，作者也写了读取函数，不过好像要翻历史版本，现在提供的是又一次修改后的模型
> 
> 好滴，非常感谢，我先看看

权重文件下载到了，但翻了一圈历史的commit，没看到作者的读取函数

**Comment by YUTING0907** [comment]
> > > > 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？
> > > 
> > > 
> > > 得用预训练权重，不然没有泛化性
> > 
> > 
> > 请问预训练权重文件在哪里呀？
> 
> 作者在readme里提供了
> 
> download weights from: https://drive.google.com/drive/folders/1UpLYGLU5omztVsIWkRPFTVKAOVe_4p3K?usp=sharing
> 
> 我遇到了AVA数据集的预训练权重指标不太对的问题，如果您训练出结果可以提供一下您训练之后的指标吗，想做一个参考，感谢

在AVA上跑了20个epoch，plcc：0.61，srcc: 0.59

**Comment by NurseryRhymeA** [comment]
> > > > > 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？
> > > > 
> > > > 
> > > > 得用预训练权重，不然没有泛化性
> > > 
> > > 
> > > 请问预训练权重文件在哪里呀？
> > 
> > 
> > 作者在readme里提供了
> > download weights from: https://drive.google.com/drive/folders/1UpLYGLU5omztVsIWkRPFTVKAOVe_4p3K?usp=sharing
> > 我遇到了AVA数据集的预训练权重指标不太对的问题，如果您训练出结果可以提供一下您训练之后的指标吗，想做一个参考，感谢
> 
> 在AVA上跑了20个epoch，plcc：0.61，srcc: 0.59

谢谢！

**Comment by YUTING0907** [comment]
> > > > > > 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？
> > > > > 
> > > > > 
> > > > > 得用预训练权重，不然没有泛化性
> > > > 
> > > > 
> > > > 请问预训练权重文件在哪里呀？
> > > 
> > > 
> > > 作者在readme里提供了
> > > download weights from: https://drive.google.com/drive/folders/1UpLYGLU5omztVsIWkRPFTVKAOVe_4p3K?usp=sharing
> > > 我遇到了AVA数据集的预训练权重指标不太对的问题，如果您训练出结果可以提供一下您训练之后的指标吗，想做一个参考，感谢
> > 
> > 
> > 在AVA上跑了20个epoch，plcc：0.61，srcc: 0.59
> 
> 谢谢！

您这边训练的指标是多少呢？

**Comment by sevenotes** [comment]
> > > > > > > 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？
> > > > > > 
> > > > > > 
> > > > > > 得用预训练权重，不然没有泛化性
> > > > > 
> > > > > 
> > > > > 请问预训练权重文件在哪里呀？
> > > > 
> > > > 
> > > > 作者在readme里提供了
> > > > download weights from: https://drive.google.com/drive/folders/1UpLYGLU5omztVsIWkRPFTVKAOVe_4p3K?usp=sharing
> > > > 我遇到了AVA数据集的预训练权重指标不太对的问题，如果您训练出结果可以提供一下您训练之后的指标吗，想做一个参考，感谢
> > > 
> > > 
> > > 在AVA上跑了20个epoch，plcc：0.61，srcc: 0.59
> > 
> > 
> > 谢谢！
> 
> 您这边训练的指标是多少呢？
抱歉，我刚才找了一下没找到留档，但我记得当时用的是TAD66K，效果也是比论文指标低了一些

**Comment by NurseryRhymeA** [comment]
> > > > > > > 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？
> > > > > > 
> > > > > > 
> > > > > > 得用预训练权重，不然没有泛化性
> > > > > 
> > > > > 
> > > > > 请问预训练权重文件在哪里呀？
> > > > 
> > > > 
> > > > 作者在readme里提供了
> > > > download weights from: https://drive.google.com/drive/folders/1UpLYGLU5omztVsIWkRPFTVKAOVe_4p3K?usp=sharing
> > > > 我遇到了AVA数据集的预训练权重指标不太对的问题，如果您训练出结果可以提供一下您训练之后的指标吗，想做一个参考，感谢
> > > 
> > > 
> > > 在AVA上跑了20个epoch，plcc：0.61，srcc: 0.59
> > 
> > 
> > 谢谢！
> 
> 您这边训练的指标是多少呢？

抱歉，我刚才找了一下没找到留档，但我记得当时用的是TAD66K，效果也是比论文指标低了一些

**Comment by zhuyeqing** [comment]
> > > > > 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？
> > > > 
> > > > 
> > > > 得用预训练权重，不然没有泛化性
> > > 
> > > 
> > > 请问预训练权重文件在哪里呀？
> > 
> > 
> > 作者在readme里提供了
> > download weights from: https://drive.google.com/drive/folders/1UpLYGLU5omztVsIWkRPFTVKAOVe_4p3K?usp=sharing
> > 我遇到了AVA数据集的预训练权重指标不太对的问题，如果您训练出结果可以提供一下您训练之后的指标吗，想做一个参考，感谢
> 
> 在AVA上跑了20个epoch，plcc：0.61，srcc: 0.59

你好，我是一名pytorch的初学者，最近也在关注这个项目，想在自己的数据集上测试下，但是对这项目的代码和模型有很多不明白之处，不知道你能否分享下你的代码，多谢多谢？

**Comment by NurseryRhymeA** [comment]
> > > > > 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？
> > > > 
> > > > 
> > > > 得用预训练权重，不然没有泛化性
> > > 
> > > 
> > > 请问预训练权重文件在哪里呀？
> > 
> > 
> > 作者在readme里提供了
> > download weights from: https://drive.google.com/drive/folders/1UpLYGLU5omztVsIWkRPFTVKAOVe_4p3K?usp=sharing
> > 我遇到了AVA数据集的预训练权重指标不太对的问题，如果您训练出结果可以提供一下您训练之后的指标吗，想做一个参考，感谢
> 
> 在AVA上跑了20个epoch，plcc：0.61，srcc: 0.59

最近在tad66k上跑了几个30epoch的实验，都是plcc在0.51左右，srcc在0.48左右.  论文中给出的是0.546和0.517

**Comment by zhuyeqing** [comment]
> > > > > > 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？
> > > > > 
> > > > > 
> > > > > 得用预训练权重，不然没有泛化性
> > > > 
> > > > 
> > > > 请问预训练权重文件在哪里呀？
> > > 
> > > 
> > > 作者在readme里提供了
> > > download weights from: https://drive.google.com/drive/folders/1UpLYGLU5omztVsIWkRPFTVKAOVe_4p3K?usp=sharing
> > > 我遇到了AVA数据集的预训练权重指标不太对的问题，如果您训练出结果可以提供一下您训练之后的指标吗，想做一个参考，感谢
> > 
> > 
> > 在AVA上跑了20个epoch，plcc：0.61，srcc: 0.59
> 
> 最近在tad66k上跑了几个30epoch的实验，都是plcc在0.51左右，srcc在0.48左右. 论文中给出的是0.546和0.517

你好，我是一名pytorch的初学者，最近也在关注这个项目，想在自己的数据集上测试下，但是对这项目的代码和模型有很多不明白之处，不知道你能否分享下你的代码，多谢多谢？

**Comment by NurseryRhymeA** [comment]
> > > > > > > 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？
> > > > > > 
> > > > > > 
> > > > > > 得用预训练权重，不然没有泛化性
> > > > > 
> > > > > 
> > > > > 请问预训练权重文件在哪里呀？
> > > > 
> > > > 
> > > > 作者在readme里提供了
> > > > download weights from: https://drive.google.com/drive/folders/1UpLYGLU5omztVsIWkRPFTVKAOVe_4p3K?usp=sharing
> > > > 我遇到了AVA数据集的预训练权重指标不太对的问题，如果您训练出结果可以提供一下您训练之后的指标吗，想做一个参考，感谢
> > > 
> > > 
> > > 在AVA上跑了20个epoch，plcc：0.61，srcc: 0.59
> > 
> > 
> > 最近在tad66k上跑了几个30epoch的实验，都是plcc在0.51左右，srcc在0.48左右. 论文中给出的是0.546和0.517
> 
> 你好，我是一名pytorch的初学者，最近也在关注这个项目，想在自己的数据集上测试下，但是对这项目的代码和模型有很多不明白之处，不知道你能否分享下你的代码，多谢多谢？

我使用的就是作者上传的代码，没有修改，数据集下载下来就可以运行了

**Comment by sid-022** [comment]
> > > > > > > > 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？
> > > > > > > 
> > > > > > > 
> > > > > > > 得用预训练权重，不然没有泛化性
> > > > > > 
> > > > > > 
> > > > > > 请问预训练权重文件在哪里呀？
> > > > > 
> > > > > 
> > > > > 作者在readme里提供了
> > > > > download weights from: https://drive.google.com/drive/folders/1UpLYGLU5omztVsIWkRPFTVKAOVe_4p3K?usp=sharing
> > > > > 我遇到了AVA数据集的预训练权重指标不太对的问题，如果您训练出结果可以提供一下您训练之后的指标吗，想做一个参考，感谢
> > > > 
> > > > 
> > > > 在AVA上跑了20个epoch，plcc：0.61，srcc: 0.59
> > > 
> > > 
> > > 最近在tad66k上跑了几个30epoch的实验，都是plcc在0.51左右，srcc在0.48左右. 论文中给出的是0.546和0.517
> > 
> > 
> > 你好，我是一名pytorch的初学者，最近也在关注这个项目，想在自己的数据集上测试下，但是对这项目的代码和模型有很多不明白之处，不知道你能否分享下你的代码，多谢多谢？
> 
> 我使用的就是作者上传的代码，没有修改，数据集下载下来就可以运行了

请问这边训练好的模型大小大概是多少呢？

**Comment by 31b14fee** [comment]
> > > > > 您好，我按照默认参数训练了20个epoch，同样lcc只有0.10，请问您是怎样解决这个问题的？
> > > > 
> > > > 
> > > > 得用预训练权重，不然没有泛化性
> > > 
> > > 
> > > 请问预训练权重文件在哪里呀？
> > 
> > 
> > 作者在readme里提供了
> > download weights from: https://drive.google.com/drive/folders/1UpLYGLU5omztVsIWkRPFTVKAOVe_4p3K?usp=sharing
> > 我遇到了AVA数据集的预训练权重指标不太对的问题，如果您训练出结果可以提供一下您训练之后的指标吗，想做一个参考，感谢
> 
> 在AVA上跑了20个epoch，plcc：0.61，srcc: 0.59

你好，请问你解决这个问题了吗？我将超参数设置为epoch：30 batch size：8 learning rate：1e-5，AVA的PLCC：0.737 SRCC：0.722和论文中的指标还是有较大差距。而且我发现在dat_blocks.py中对兴趣点做了batch的平均池化，我在调参的过程中也发现batch size越小性能越好。是因为平均池化导致的性能下降吗？
